app/src/debug/java/org/simple/clinic/remoteconfig/firebase/FirebaseRemoteConfigModule.kt

```diff
@@ -15,23 +16,14 @@ class FirebaseRemoteConfigModule {
   @AppScope
   fun remoteConfig(): FirebaseRemoteConfig {
     return FirebaseRemoteConfig.getInstance().apply {
-      // Enable developer mode so that calls are not throttled during dev.
-      // More details in FirebaseRemoteConfigCacheExpiration.
-      setConfigSettings(FirebaseRemoteConfigSettings.Builder()
-          .setDeveloperModeEnabled(true)
+      val settings = FirebaseRemoteConfigSettings
+          .Builder()
+          .setMinimumFetchIntervalInSeconds(Duration.ofMinutes(1).seconds)
```

Comment:
Let's change it to this just as it was before?
`Duration.ofMinutes(0)`

Reply:
Will do.